parameter/ConfigurableElement.cpp

```diff
@@ -155,6 +157,54 @@ bool CConfigurableElement::accessValue(CPathNavigator& pathNavigator, std::strin
     return pChild->accessValue(pathNavigator, strValue, bSet, parameterAccessContext);
 }
 
+// Whole element access
+void CConfigurableElement::getSettingsAsBytes(std::vector<uint8_t>& bytes,
+                                              CParameterAccessContext& parameterAccessContext) const
+{
+    CParameterBlackboard* pParameterBlackboard = parameterAccessContext.getParameterBlackboard();
+
+    bytes.reserve(getFootPrint());
+
+    pParameterBlackboard->readBytes(bytes, getOffset() - parameterAccessContext.getBaseOffset());
```

Comment:
the pParameterBlackboard blackboard temporary variable does not seem very usefull.